### PR TITLE
Record provider errors 184923964

### PR DESCRIPTION
**Pull request by vrajmohan**

### CODE REVIEW

File: src/main/java/org/codeforamerica/messaging/models/SmsMessage.java [hunk 1]
```diff
@@ -43,6 +46,6 @@ public class SmsMessage {
     @JsonIgnore
     @OneToOne(mappedBy = "smsMessage")
     private Message message;
+    @JdbcTypeCode( SqlTypes.JSON)
```
Comment (mpidcock) [inline src/main/java/org/codeforamerica/messaging/models/SmsMessage.java:49]: Extra space, both here and in the matching line of `EmailMessage`.

I'll do a real review next week.

Comment (vrajmohan) [reply src/main/java/org/codeforamerica/messaging/models/SmsMessage.java:49]: Thanks, this is getting to be embarrassing.

Comment (mpidcock) [reply src/main/java/org/codeforamerica/messaging/models/SmsMessage.java]: The minor errors don't matter much and make me feel slightly better when I also miss things :)

**Comment by mpidcock** [comment]
Just taking note of how annoying it is that to me the concepts of `errorMessage`, `errorDescription`, and `reason` are all redundant/duplicate, but without full understanding of what kinds of values appear in each, I don't yet feel comfortable pruning or mapping them to a new structure, though I can imagine we will end up doing that eventually.

**Review by mpidcock — APPROVED** [review]

**Comment by vrajmohan** [comment]
> Just taking note of how annoying it is that to me the concepts of `errorMessage`, `errorDescription`, and `reason` are all redundant/duplicate, but without full understanding of what kinds of values appear in each, I don't yet feel comfortable pruning or mapping them to a new structure, though I can imagine we will end up doing that eventually.

Yes, I blame it on sloppy design from `Mailgun`. Their documentation also smacks of poor maintenance.

**Pull request merged**